internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -55,7 +56,8 @@ import (
 const (
 	adminPasswordSecretStringTemplate = "%s-admin-password"
 	adminKubeConfigStringTemplate     = "%s-admin-kubeconfig"
-	InstallConfigOverrides            = v1beta1.Group + "/install-config-overrides"
+	InstallConfigOverrides            = aiv1beta1.Group + "/install-config-overrides"
+	ClusterDeploymentFinalizerName    = "clusterdeployments.hive.openshift.io/finalizer"
```

Comment:
Doesn't it defined in Hive? can't you import this definition?

Reply:
Looked it up and I could not find it in Hive.
@dgoodwin, did I miss it?

Reply:
You'll want to use your own finalizer, as we would never want two controllers that might be trying to remove the same finalizer. Bear in mind that this finalizer is about to move to the AgentClusterInstall once Fred's work merges.

I'd do something like agent-install.openshift.io/deprovision